docs/crio.8.md

```diff
@@ -212,6 +213,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--enable-tracing**: Enable OpenTelemetry trace data exporting
 
+**--evented-pleg**: Whether CRI-O should generate the container events in order to optimize the performance of Kubelet to rely on these events.
```

Comment:
I'm not terribly familiar with this option, and this description is not helping whatsoever.  Plus, is it a boolean?  If you make changes here, please also change in crio.fish.

Maybe?
```suggestion
**--evented-pleg**: Enable CRI-O to generate the container pod-level events in order to optimize the performance of the Pod Lifecycle Event Generator (PLEG) module in Kubelet.
```

Reply:
Thanks Tom, Noted.